components/ResultsPageComponents/SurveyOverview/SurveyOverview.jsx

```diff
@@ -6,7 +6,51 @@ import DottedLine from "../../../public/images/survey-info-icons/dotted-line.svg
 import Path from "../../../public/images/survey-info-icons/path.svg";
 import Respondents from "../../../public/images/survey-info-icons/respondents.svg";
 
-export default function SurveyOverview() {
+const getMonthName = (month) => {
+  switch (month) {
+    case 0:
+      return "January";
+    case 1:
+      return "February";
+    case 2:
+      return "March";
+    case 3:
+      return "April";
+    case 4:
+      return "May";
+    case 5:
+      return "June";
+    case 6:
+      return "July";
+    case 7:
+      return "August";
+    case 8:
+      return "September";
+    case 9:
+      return "October";
+    case 10:
+      return "November";
+    case 11:
+      return "December";
+    default:
+      return "???";
```

Comment:
Should we leave it empty instead of "???"? Or call it "of this month" so doesn't look like we have a bug if this happens. ))

Reply:
Good point @OlaShabalina !
Doesn't look good for a report to have a bunch of '???' 😓 

I'll fix it 👍🏽